opencsp/common/lib/csp/OpticOrientationAbstract.py

```diff
@@ -156,31 +152,28 @@ def get_transform_relative_to(self: 'OpticOrientationAbstract', target: 'OpticOr
         `TransformXYZ`
             The transformation that takes self to the `target` frame of reference
         """
-        warn("this function has not been verified")
         if target is self:
             return TransformXYZ.identity()
 
         # look up the tree
         searcher = self
         transform = TransformXYZ.identity()
-        while searcher._self_to_parent_transform is not None:
-            if searcher is target:
-                return transform
+        while searcher.parent is not None:
```

Comment:
consider filing a follow-on issue if these linear searches are expected to be a bottleneck

Reply:
Sounds good. As we use this capability more, I will keep this in mind.